filament/backend/CMakeLists.txt

```diff
@@ -235,8 +235,8 @@ function(build_vkshader SOURCE TARGET_PATH)
             DEPENDS matc ${SOURCE_PATH}
             COMMENT "Building SPIR-V")
 endfunction()
-build_vkshader("BlitColor.vs" "BlitColorVs.spv")
-build_vkshader("BlitColor.fs" "BlitColorFs.spv")
+build_vkshader("BlitDepth.vs" "BlitDepthVs.spv")
+build_vkshader("BlitDepth.fs" "BlitDepthFs.spv")
```

Comment:
Do we smol-v compress shaders built outside of materials?

Reply:
That's an interesting idea, currently our compileRawShader function does _not_ apply smolv compression...

Reply:
Let me give this a shot.

Reply:
@romainguy I implemented your idea and added a commit.